### PR TITLE
refreshing for debug

### PR DESCRIPTION
**Pull request by d-a-v**
- Removed old buffered debug-print
- Due to newlib update for esp8266-Arduino, use `printf` instead of `os_printf`

### CODE REVIEW

**Comment by mcspr** [comment]
make -f Makefile.arduino all now throws this
```
../../glue/gluedebug.h:59:10: fatal error: osapi.h: No such file or directory
   59 | #include "osapi.h" // os_printf* definitions + ICACHE_RODATA_ATTR
      |          ^~~~~~~~~
```

ok building with cmake with edited header, but without the buffer it seems to overlap some data and I never see anything starting with `lwESP`

**Comment by mcspr** [comment]
re. missing lwESP, those show up after enabling debug flags like esp port and at least core level. Does not seem consistent :)

File: glue/gluedebug.h [hunk 1]
```diff
@@ -65,41 +55,21 @@ extern int doprint_allow;
...
-#endif // !STRING_IN_FLASH
-
-int doprint_minus (const char* format, ...) __attribute__ ((format (printf, 1, 2))); // format in flash
-
-#else // !( UDEBUG && (UDEBUGINDEX || UDEBUGSTORE) )
-
-#define doprint(x...)		do { os_printf(x); } while (0)
-
-#endif // !( UDEBUG && (UDEBUGINDEX || UDEBUGSTORE) )
+#if defined(ARDUINO) && !defined(LWIP14GLUE)
```
Comment (mcspr) [inline glue/gluedebug.h:61]: shouldn't the `LWIP14GLUE` be applied to *everything* we build and *always* use newlib printf? i.e. 'this section only applies when we build glue, not when we include lwip headers later' (that are inconveniently chained through cc.h)
we have inconsistent `os_printf` that needs both Serial begin and debug enabled vs. `printf` that always works after Serial begin

Comment (d-a-v) [reply glue/gluedebug.h]: Done

**Comment by mcspr** [comment]
I may be misremembering the whole set of issues... Wasn't there also some kind of problem with the `#include` order?
Meaning, lwip cc and etc .h set a lot of stuff for glue to build in lwip1.4 compatible mode. But, is it generally safe to keep those things in exported headers?

**Comment by d-a-v** [comment]
The purpose has always been to unexpose lwip1.4 to upper layers (Arduino).
The lwip1.4 side is only useful to link with and talk to nonosdk libs.
This PR intention is to cleanup unused file (I had made them because I was not aware that my usbserial chip was unable to to use 74880 back then)

**Pull request merged**